data/remnant/remnant outfits.txt

```diff
@@ -276,6 +277,136 @@ outfit "EMP Torpedo Bay"
 		"missile strength" 80
 	description "Electromagnetic pulse weapons were developed by the Remnant during the early days of their colony, when they were living in fear that the Alphas would overrun human space and expand beyond it."
 
+
+
+outfit "Teciimach Canister"
+	plural "Teciimach Canisters"
+	category "Ammunition"
+	licenses
+		Remnant
+	cost 5500
+	thumbnail "outfit/emp torpedo"
+	"mass" 1
+	"teciimach canister capacity" -1
+	description "This canister contains everything required to assemble a Teciimach torpedo. These weapons emit strong electromagnetic pulses which interfere with a ship's generators and neutralize a portion of their energy."
+	description `	Unlike the previous generation of "EMP torpedoes," the Teciimach torpedoes actively shape their emission pattern to avoid impacting their allies.`
+
+outfit "Teciimach Canister Rack"
+	plural "Teciimach Canister Racks"
+	category "Ammunition"
+	licenses
+		Remnant
+	cost 98000
+	thumbnail "outfit/emp rack"
+	"mass" 5
+	"outfit space" -9
+	"energy capacity" 500
+	"teciimach canister capacity" 9
+	ammo "Teciimach Canister"
+	description "The Teciimach Canister rack is used to store extra sets of components, both to expand the capacity of a ship, and on carriers for resupplying fighters."
+	description "	In addition to its ammunition capacity, the storage bay can also contain a small amount of charge, enough to let an otherwise battery-less ship assemble Teciimach torpedoes."
+
+outfit "Teciimach Bay"
+	plural "Teciimach Bays"
+	category "Secondary Weapons"
+	licenses
+		Remnant
+	cost 583000
+	thumbnail "outfit/teciimach bay"
+	"mass" 9
+	"outfit space" -13
+	"weapon capacity" -13
```

Comment:
13 space and 9 ammo would imply a mass of 4. But I'll note how the Teciimach bay deals about 80% more energy drain (taking ionization and energy damages into account together), 30% more shield damage, and 25% more hull damage, and has 25% better turning, about 12% or 14% better tracking (depending on the tracking type), 19% better missile strength, and being a "safe" weapon, all while being about 70% the size, having the exact same bay cost and capacity, and only having the ammunition cost 25% more.

I would expect a weapon with this many additional benefits to either have a much larger and more expensive launcher with ammunition that's a bit more expensive than just +25%, or if it is to be made smaller then for it to have a reduced capacity, say 6 capacity with a mass of 7, with much more expensive ammunition, say 25k at the very least, but I'd be tempted to go above even that. (Note that even at a capacity of 6, a Teciimach would still output more total ion+energy damage than a normal EMP bay by about 18% in about 60% of the time.)

Reply:
Thank  you for pointing out the prices - I'd totally neglected to change those. I have fixed that now.

In terms of explanation, the prices are fairly massive increases. The main launcher is almost a tenfold increase in cost, and the ammo is a five fold increase. The pod is about half the price of the launcher. At this point in the Remnant story, the player has complete a fairly long string of missions that pay decently; and have access to jobs that pay quite well. They also have extremely easy access to farmable/lootable enemy ships that are an easy source of millions of credits. 

So on a meta level, this is both a balance to the easy loot economy that players at this stage have access to.

On the lore side, this has the triple hit of:
- being cutting edge tech that is upgrading their centuries old stuff.
- being a smaller economy that doesn't have the advanges of the massive mass production that, say, the Wanderers or Coalition can do, or even the Korath.
- being brand new, and thus higher prices of stuff just reaching distribution.